src/util/fileToPng.ts

```diff
@@ -4,6 +4,7 @@ import {PNG} from 'pngjs';
 /**
  * Given a path to a PNG, returns a pngjs object
  */
+// eslint-disable-next-line
```

Comment:
Is this needed? Can we add the specific rule to disable?

Reply:
I will just remove this file entirely, I think it's unused